src/Components/RosTable/DiskUsage.js

```diff
@@ -0,0 +1,48 @@
+import { Tooltip } from '@patternfly/react-core';
+import React from 'react';
+import { NO_DATA_STATE, NO_DATA_VALUE } from '../../constants';
+import './DiskUsage.scss';
+
+export const diskUsageTitle = () =>{
+    return (
+        <Tooltip content={<span>IOPS</span>}>
+            <span>I/O utilization</span>
+        </Tooltip>
+    );
+};
+
+export const DiskUsageData = (data, id, item) => {
+    const { state, performance_utilization: performanceUtilization } = item;
+    const { io_all: iopsAll } = performanceUtilization;
+
+    return (
+        state === NO_DATA_STATE ?
+            <span>{ NO_DATA_VALUE }</span> :
+            <Tooltip position="right" content={
+                <div>
+                    <table>
+                        <tr>
+                            <th>Device name</th>
+                            <th>Value</th>
+                        </tr>
+                        <tr>
+                            <td colSpan="100%" className="seperator"></td>
+                        </tr>
+                        {
+                            Object.keys(iopsAll).map((deviceName, index) =>{
+                                return (
+                                    <tr key={index}>
+                                        <td>{deviceName}</td>
+                                        <td>{iopsAll[deviceName]}</td>
+                                        <td>IOPS</td>
```

Comment:
IOPS is a unit for values so will it be possible to club last two columns in one?

Reply:
@kgaikwad Since based on [mockup](https://marvelapp.com/prototype/geig8bh/screen/82194253) I felt that the `IOPS` unit should be aligned vertically and hence to achieve that I added it as seperate column. Let me know if this is okay else I will have to check how to achieve the same in two columns.

Reply:
Yes, I guess with content alignment that can be achievable but I will also check. For now, let's go with current implementation.